packages/resin-event-log/src/resin-event-log.js

```diff
@@ -61,14 +61,10 @@ module.exports = function(options) {
 		userId: null,
 		prefix: prefix,
 		start: function(user, callback) {
-			if (!user) {
-				return Promise.reject(new Error(
-					'user is required to start events interaction.'
-				)).asCallback(callback)
```

Comment:
Ok, sure. In that case, this part of the approach looks broadly fine to me, but it needs extending to the rest, and checking that that doesn't break anything, before we can merge it.